sentry_sdk/integrations/asgi.py

```diff
@@ -57,17 +57,6 @@
 TRANSACTION_STYLE_VALUES = ("endpoint", "url")
 
 
-def _capture_exception(exc, mechanism_type="asgi"):
```

Comment:
I would prefer to keep this function the same, as a standalone function instead of a method. I don't think it has to be modified to implement your changes